src/JunctionTreeTypes.jl

```diff
@@ -51,6 +54,8 @@ mutable struct CliqStateMachineContainer{BTND, T <: AbstractDFG, InMemG <: InMem
   refactoring::Dict{Symbol, String}
   oldcliqdata::BTND
   logger::SimpleLogger
+  parametricMsgsUp::Vector{ParametricBeliefMessage} #TODO TEMP net om te kyk hoe dit werk, dit moet eintelik <: abstact wees
+  parametricMsgsDown::Vector{ParametricBeliefMessage}
```

Comment:
Again, I am quite worried about the <:Abstract concept being introduced everywhere.  There was the long discussion on anonymous static types, where the Julia guys were nervous.  We let it be `estimateDict` case, but I don't think we should be introducing it everywhere so quickly.  As with the comment above, I feel there should be a single BeliefMessage type that works for all cases -- although this will have to happen as part of tree message consolidation.

Reply:
<: abstract parametric, was to allow for different message types, I believe one message type will work.